src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -383,13 +383,21 @@ class MediaBaseViewer extends BaseViewer {
      * @emits ratechange
      * @return {void}
      */
-    handleRate() {
+    handleRate(rate) {
+        if (rate) {
```

Comment:
It looks like this will affect the existing controls. Can we de-risk this change by creating separate handlers for our new controls?